src/Fixer/FunctionNotation/PhpdocToReturnTypeFixer.php

```diff
@@ -251,6 +251,12 @@ protected function applyFix(\SplFileInfo $file, Tokens $tokens)
                 continue;
             }
 
+            try {
+                Tokens::fromCode(sprintf('<?php function f():%s {}', $returnType));
```

Comment:
I suggest to already implement a caching system, see https://github.com/kubawerlos/PHP-CS-Fixer/pull/4